vpr/src/draw/draw_triangle.cpp

```diff
@@ -0,0 +1,122 @@
+/* draw_triangle.cpp contains functions that draw triangles.*/
```

Comment:
Can you say what they are used for?

Reply:
(They are generally used for showing switching in the routing, the direction of signals, and such.)